components/hybrid_engine/src/observer.rs

```diff
@@ -13,14 +13,18 @@ use raftstore::coprocessor::{
 
 #[derive(Clone)]
 pub struct Observer {
-    pending_evict: Arc<Mutex<Vec<(CacheRange, EvictReason)>>>,
+    // observer is per thread so ther is no need to use mutex here,
```

Comment:
```suggestion
    // observer is per thread so ther is no need to use mutex here,
```// observer is per thread so their is no need to use mutex here,